Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -90,7 +90,7 @@ def account_entry(person_object, custome_attributes):
         'Email': person_object.get('email'),
         'Username': person_object.get('samAccountName'),
         'DisplayName': person_object.get('displayName'),
-        'Managr': person_object.get('manager'),
+        'Manager': person_object.get('manager'),
```

Comment:
we need to maintain the old one as well, so keep the old version and also add the new one.
Also please verify the outputs

Reply:
@Shellyber what about this?